brianlong/www.validators.app
# 182342726-fix-links-in-emails

#### What's this PR do?
- setup devise emails on stage

#### How should this be manually tested?
- on stage try to register, remind password etc. Check if all links redirect to staging server

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182342726)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
